rialto_airflow/google.py

```diff
@@ -8,8 +8,48 @@
 from airflow.providers.google.suite.hooks.sheets import GSheetsHook
 from airflow.providers.google.suite.hooks.drive import GoogleDriveHook
 from airflow.models import Variable
+import dotenv
+
+# Load environment variables
+dotenv.load_dotenv()
 
 gcp_conn_id = Variable.get("google_connection", "google_cloud_default")
+google_drive_id = Variable.get("google_drive_id")
+
+
+# The Google Drive folder ID for the open access dashboard
+def open_access_dashboard_folder_id():
+    return get_file_id(google_drive_id, "open-access-dashboard")
+
+
+# The Google Drive folder ID for the ORCID dashboard
+def orcid_dashboard_folder_id():
+    return get_file_id(google_drive_id, "orcid-dashboard")
+
```

Comment:
as we add more dashboard folders in the shared google drive, they will get added here, so we can make them available to the dags